### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by Tristramg** [etalab/transport-topo#5]
As the tool is heavily dependent on an instance of wikimedia and its api and sparql service, integration tests are probably the most important.

This requires :

- [ ] launch a docker instance of all services (can we user the docker-compose as basis?)
- [ ] populate it with the basic identities and properties (can it be included in the docker ?)
- [ ] test individual functions (search…)
- [ ] end-to-end test (idempotence of importing a gtfs file…)

**Comment by antoine-de** [comment]
seems great!

For the technical part, I've tried https://github.com/testcontainers on a project. It was simple (no dependency to docker-compose), but not that bad, apart from some bad container cleaning in some cases.

Else we've implemented [one wrapper](https://github.com/CanalTP/mimirsbrunn/blob/master/libs/docker_wrapper/src/lib.rs) for mimir by hand, it's not very complicated.

I've seen integration tests in python using docker-compose and It seems real nice, but I've not seen anything like this in rust 😢 

How many container are needed? And do they depend on each other?
